Formula/databend.rb

```diff
@@ -0,0 +1,28 @@
+class Databend < Formula
+  desc "Elastic, reliable serverless data warehouse, offers blazing fast query"
+  homepage "https://databend.rs/"
+  url "https://github.com/datafuselabs/databend/archive/refs/tags/v0.6.36-nightly.tar.gz"
+  sha256 "73c4957bd51745b4bacd16f6b7324e14f74b68b1ff53b165ad780e8b23667876"
+  license "Apache-2.0"
+
+  livecheck do
+    url :url
+  end
+
+  depends_on "rustup-init" => :build
```

Comment:
This should depend on `rust`. We do have a handful of formulae that use `rustup`, but those are exceptions since they were needed by existing formulae. New formulae shouldn't need `rustup`.

Reply:
but we need nightly version for rust which rustup can deal with well.

Reply:
That's another reason that makes this unsuitable for packaging in Homebrew.

Reply:
Ok I get it, thx for nice explaining.